ui/src/institutions/containers/DetailPageContainer.jsx

```diff
@@ -57,7 +57,7 @@ function DetailPage({ record, isSuperUserLoggedIn }) {
   return (
     <>
       <DocumentHead title={legacyIcn} description={metaDescription} />
-      <Row justify="center">
+      <Row justify="center" data-testid="detail-page-container">
```

Comment:
lets prefix the test ids with the collection so as to ensure no overlapping. Something like 

institutions-detail-page-container. 

Also since we are doing that, could you please update the id for the Data detail to 

data-detail-page-container ?